e2e/framework/test_context.go

```diff
@@ -149,11 +150,13 @@ type TestContextType struct {
 	// Wizard specifies whether wizard was used to bootstrap cluster
 	Wizard bool `json:"-" yaml:"-"`
 	// Provisioner defines the type of provisioner to use
-	Provisioner provisionerType `json:"provisioner" yaml:"provisioner" env:"ROBO_PROVISIONER"`
+	Provisioner provisionerType `json:"provisioner" yaml:"provisioner" `
+	// ProvisionTo defines cloud to deploy
+	ProvisionTo string `json:"provision_to" yaml:"provision_to" validate:"omitempty,eq=aws|eq=azure"`
 	// DumpCore specifies a command to collect all installation/operation logs
 	DumpCore bool `json:"-" yaml:"-"`
 	// StateDir specifies the location for test-specific temporary data
-	StateDir string `json:"state_dir" yaml:"state_dir" env:"ROBO_STATE_DIR"`
```

Comment:
https://docs.google.com/document/d/1lwbmjakDVan8haGwHUX65r5OR15tTSiFsz6qKKNVt5k/edit

just like we're getting rid of static variables; to allow concurrent multi-suite tests